packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -21,8 +21,8 @@ const { initializeProgressBars } = require('./aws-cfn-progress-formatter');
 const { printer } = require('amplify-prompts');
 
 const CFN_MAX_CONCURRENT_REQUEST = 5;
-const CFN_POLL_TIME = 5 * 1000; // 5 secs wait to check if  new stacks are created by root stack
-const CFN_POLL_TIME_MAX = 30 * 1000; // 30 seconds
+const CFN_POLL_TIME = (process.env.IS_AMPLIFY_CI ? 30 : 5) * 1000; // 5 secs wait to check if  new stacks are created by root stack
+const CFN_POLL_TIME_MAX = (process.env.IS_AMPLIFY_CI ? 120 : 30) * 1000; // 30 seconds
```

Comment:
still experimenting with this, tests were failing on some batches where multiple long running deployments occur;
it seems to have fixed most of the throttling

Reply:
Hoping to follow up on this:
1. create a follow up PR that fixes “too many requests” exceptions caused by CFN polling but maintains the same logic on E2E as w/ regular customers
- get rid of custom polling rate for e2e
- implement error handling for “too many requests” instead

result: 
logic is the same for both customers & our e2e tests